yb-voyager/src/query/queryissue/constants.go

```diff
@@ -457,13 +457,13 @@ Note: If the table is created as colocated, this hotspot concern can safely be i
 
 	NULL_VALUE_INDEXES_ISSUE_NAME                = "Index on column with a high percentage of NULL values"
 	NULL_VALUE_INDEXES                           = "INDEX_ON_COLUMN_WITH_HIGH_PERCENTAGE_OF_NULL_VALUES"
-	NULL_VALUE_INDEXES_DESCRIPTION               = "In distributed databases, index design should ensure even data distribution across nodes. Indexes on columns with many NULL values can lead to uneven distribution and can cause performance issues."
+	NULL_VALUE_INDEXES_DESCRIPTION               = "In distributed databases, unnecessary index entries for NULL values can increase write performance and storage usage. If queries do not filter on NULL values, using partial indexes can eliminate these unnecessary writes."
```

Comment:
Slight reword (same applies for the other issue as well): (just want to mention the problem first and then explain why it would be more important to fix in a distributed DB)

```This index contains a significant percentage of NULL values. Indexes with many NULL values can impact performance and storage efficiency, more so in distributed databases like YugabyteDB. 

if your queries don't need to filter on NULL values, consider creating a partial index that excludes NULL values.
```